### PR TITLE
Customizable login branding + UI redesign

### PR DESCRIPTION
**Pull request by timotree3**
This PR updates Chaperone UI to have customizable branding and a new look and feel as specified in the designs.

### CODE REVIEW

**Comment by timotree3** [comment]
image


This is ready for review!

**Comment by timotree3** [comment]
If you want to test this out, I recommend checking out my EC UI branch holochain/elemental-chat-ui#51

**Comment by timotree3** [comment]
Note for reviewer: The reason that the SVGs are not referenced by URL even though they are used many times is that if I put them in the `dist/` folder, they are served with the wrong Content-Type by `chaperone-server` such that they render as text not images.

File: src/static/index.html [hunk 1]
```diff
@@ -1,249 +1,752 @@
...
-	      }
+	    }
 
+		const url_params = new URLSearchParams(window.location.search);
+	    if (url_params.has('app_name')) {
+		  $(".app-name").text(url_params.get('app_name'));
+		}
+		if (url_params.has('logo_url')) {
+    	  $(".app-logo").attr("src", url_params.get('logo_url'));
+		}
```
Comment (JettTech) [inline src/static/index.html:671]:
```suggestion
		if (url_params.has('logo_url')) {
    	  $(".app-logo").attr("src", url_params.get('logo_url'));
		} else {
		  $(".app-logo").hide();
		};
```

Comment (JettTech) [reply src/static/index.html:671]: According to the designs, the logo is optional and should default to no image display. Let's just hide the img tag whenever the logo_url is nullish.

**Review by JettTech — CHANGES_REQUESTED** [review]
This looks awesome!  I'm impressed with your attention to the design details :tada:

I found one small edit to make, but otherwise, this looks good. Once this is updated, I say let's ship this!  Let me know if you have any Qs in the meantime. Almost there! :running_man:

**Comment by timotree3** [comment]
Feedback should be addressed. I tested the no logo case locally and it hides as expected.

**Review by JettTech — APPROVED** [review]
Tested and LGTM!

**Pull request merged**